OloEngine/src/OloEngine/Memory/UnrealMemory.cpp

```
@@ -4,6 +4,9 @@
 #include "OloEngine/Memory/UnrealMemory.h"
 #include "OloEngine/Memory/Memory.h"
 #include "OloEngine/Memory/GenericPlatformMemory.h"
+#include "OloEngine/HAL/MallocPoisonProxy.h"
```

Cppcheck (2.18.0)

[information] Include file

(missingInclude)